src/main/java/samza/examples/wikipedia/application/WikipediaZkLocalApplication.java

```diff
@@ -32,7 +32,15 @@
  */
 public class WikipediaZkLocalApplication {
 
-  public static void main(String[] args) throws Exception {
+  /**
+   * Executes the application using the local application runner.
+   * It takes two required command line arguments
+   *  config-factory: a fully {@link org.apache.samza.config.factories.PropertiesConfigFactory} class name
+   *  config-path: path to application properties
+   *
+   * @param args command line arguments
+   */
+  public static void main(String[] args) {
     CommandLine cmdLine = new CommandLine();
     OptionSet options = cmdLine.parser().parse(args);
     Config orgConfig = cmdLine.loadConfig(options);
```

Comment:
Do we need to rewrite config? (esp. given that there are no rewriters configured)